code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -510,6 +520,7 @@
 	color = "#CF3600"
 	metabolization_rate = 0.1
 	penetrates_skin = 1
+	taste_message = "almonds"
```

Comment:
Cyanide smells and tastes faintly of bitter-ish almonds.
Don't ask me why I know this.